scripts/sendpayment.py

```diff
@@ -54,19 +54,29 @@ def main():
         parser.error("PayJoin requires exactly three arguments: "
                      "wallet, amount and destination address.")
         sys.exit(EXIT_ARGERROR)
-    elif options.schedule == '' and len(args) != 3:
-        parser.error("Joinmarket sendpayment (coinjoin) needs arguments:"
-        " wallet, amount and destination address")
-        sys.exit(EXIT_ARGERROR)
+    elif options.schedule == '':
+        if len(args) < 2 or (not btc.is_bip21_uri(args[1]) and len(args) != 3):
```

Comment:
Perhaps be stricter? Here you disallow (a) less than 2 and (b) (not 3 and not bip21), but you should also disallow (not 2 and bip21) right?

Reply:
Yes, you're right, changed this check!